src/hdf5plugin/test.py

```diff
@@ -70,13 +73,17 @@ def tearDownClass(cls):
         shutil.rmtree(cls.tempdir)
 
     def _test(
-        self, filter_name, dtype=numpy.int32, lossless=True, compressed=True, **options
+        self,
+        filter_name: str,
+        dtype: DTypeLike = numpy.int32,
+        lossless: bool = True,
+        compressed: str | bool = True,
+        options: dict[str, Any] = None,
```

Comment:
Changing from `**options` to `options=options` for typing, but it also makes it clear what are the compression options